FFCDemoPaymentService/Data/ApplicationDbContext.cs

```diff
@@ -1,17 +1,27 @@
 using Microsoft.EntityFrameworkCore;
 using FFCDemoPaymentService.Models;
+using System.Threading.Tasks;
 
 namespace FFCDemoPaymentService.Data
 {
     public class ApplicationDbContext : DbContext
     {
-        public ApplicationDbContext() {}
-        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
-            : base(options)
+        private readonly ConnectionStringBuilder builder;
+
+        public ApplicationDbContext(ConnectionStringBuilder builder)
+            : base()
+        {
+            this.builder = builder;
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
+            string connectionString = Task.Run(builder.GetConnectionString).Result;
+            System.Console.WriteLine($"Using connection string ending: {connectionString.Substring(connectionString.Length - 10)}");
```

Comment:
We can probably remove this now unless you feel it is useful to help us monitor in the short term?

Reply:
Done. It logs every 10s, so too much to be left in!